### PR TITLE
Add to console details to "How to launch `napari`"

### PR DESCRIPTION
**Pull request by lucyleeow**
# Description

* Adds cross references to sections
* Adds note on how to open ipython console when opening napari from command line and script

### CODE REVIEW

**Review by melissawm — APPROVED** [review]
Thanks @lucyleeow !

File: docs/tutorials/fundamentals/getting_started.md [hunk 1]
```diff
@@ -47,14 +49,19 @@ If the image is `RGB` or `RGBA` use the `-r` or `--rgb` flag.
...
 ![image: napari viewer displaying an image layer](../assets/tutorials/launch_cli_image.png)
 
-Launching napari directly from the command line is the simplest and fastest way to open the viewer,
+Launching `napari` directly from the command line is the simplest and fastest way to open the viewer,
 but it doesn't allow you to preprocess your images before opening them.
 It is also currently not possible to save images or other layer types directly from the viewer,
 but we'll be adding support for this functionality soon as discussed in [#379](https://github.com/napari/napari/issues/379).
 
+If you wish to interact with your open viewer programmatically you can open an IPython
+console via **Window** > **console**.
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/getting_started.md]:
```suggestion
console via **Window** > **console** or by clicking the `Show/Hide IPython console` viewer button.
```

File: docs/tutorials/fundamentals/getting_started.md [hunk 2]
```diff
@@ -105,12 +112,17 @@ downloaded as `.py` (and `.ipynb` files) and run as above.
 
 ![image: napari launched from a python script](../assets/tutorials/launch_script.png)
 
-An advantage of launching napari from a python script
+An advantage of launching `napari` from a python script
 is that you can preprocess your images and add multiple layers before displaying the viewer.
 
+As above, if you wish to interact with your open viewer programmatically you can open
+an IPython console via **Window** > **console**.
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/getting_started.md]:
```suggestion
an IPython console via **Window** > **console** or by clicking the `Show/Hide IPython console` viewer button.
```

**Review by psobolewskiPhD — CHANGES_REQUESTED** [review]
two small suggestions

File: docs/tutorials/fundamentals/getting_started.md (hunk 1 above)
Comment (lucyleeow) [reply docs/tutorials/fundamentals/getting_started.md]: Thanks, I forgot about that. Done and I've linked to the viewer layer section!

**Review by psobolewskiPhD — APPROVED** [review]

**Comment by lucyleeow** [comment]
Thanks!

**Pull request merged**